docs/faq.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: faq
+title: FAQ
+sidebar_position: 2
+---
+
+# FAQ
+
+## What is a POAP?
+
+A [POAP](https://poap.xyz) is a digital collectible created as an NFT (non-fungible token) that represents an action taken by the owner, like a participation trophy or achievement badge. POAPs exist with cryptographic provability on the blockchain and are displayable as badges viewable by anyone.
+
+## What’s the difference between a GitPOAP and a POAP?
+
+GitPOAP is a platform built on top of the POAP protocol. GitPOAPs are regular POAPs, they are just minted through GitPOAP. They are viewable in any POAP viewing tool and have a distinctive border around them to signify that they are GitPOAPs and represent a contribution you’ve made to a project.
+
+## Are GitPOAPs soulbound?
+
+No. Although (in real life) your contributions are soulbound to you, we do not currently see the need to have GitPOAPs be soulbound to a particular address. In the future we believe people’s decentralized identities will be made up of many addresses, alongside the fact that there are justified reasons for moving tokens around. The blockchain provides transparency on how tokens are transferred so we can always find out if a GitPOAP is held by its original owner if necessary.
+
+This, however, is an area of research and focus for us and we strive to do what’s best for for the ecosystem.
+
+## Will GitPOAP be open source?
+
+Our goal is to become a trusted authority and oracle to bring off-chain contributions on chain. We are concerned that if we are open source from the start, it may dilute the quality of information coming on chain. We aim to open source as much of our platform as possible, but since there are many unknown unknowns we’ll be mostly closed source at the start.
```

Comment:
Rather than "our goal is to become ...." can you change it to "Our goal is to build a trusted record of software contributions on chain"?

I know I wrote this a while back, but after much thought, the goal really is to bring a trusted record on chain rather than becoming a trusted authority (although they are related).